components/BottomSheet/src/MDCBottomSheetController.m

```diff
@@ -73,6 +73,14 @@ - (void)viewWillAppear:(BOOL)animated {
   [self.contentViewController.view layoutIfNeeded];
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+  [super viewDidAppear:animated];
+
+  if (self.showScrollIndicatorsByDefault && self.trackingScrollView) {
```

Comment:
The `self.trackingScrollView` check here isn't necessary due to nil handling; please remove.

Reply:
Done